tests/playwright/src/model/pages/build-image-page.ts

```diff
@@ -58,10 +68,32 @@ export class BuildImagePage extends BasePage {
       await this.imageNameInput.pressSequentially(imageName, { delay: 50 });
     }
 
+    if (archType !== ArchitectureType.Default) {
+      await this.uncheckedAllCheckboxes();
+      if (archType === ArchitectureType.ARM64) {
```

Comment:
How about the rest of the architectures available? We would tackled them later? Maybe the use of a switch would be better then.

Reply:
My goal is to provide the functionality for the archs we're using right now and for which we have use cases right now, we can add more types and refactor to a switch at that point.

Reply:
refactored to use switch keyword